<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddUpdateEvents</class>
 <widget class="QWidget" name="AddUpdateEvents">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1222</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="Header">
       <property name="text">
        <string>Header</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>1200</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,2,1">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0">
        <property name="spacing">
         <number>50</number>
        </property>
        <property name="leftMargin">
         <number>50</number>
        </property>
        <property name="rightMargin">
         <number>50</number>
        </property>
        <item>
         <widget class="QFrame" name="frame_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1156</width>
            <height>520</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Shape::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Shadow::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>80</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2">
               <property name="topMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="dateLabel">
                 <property name="text">
                  <string>Дата:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="dateButton">
                 <property name="text">
                  <string>00.00.0000</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,2">
               <item>
                <widget class="QLabel" name="clientLabel">
                 <property name="text">
                  <string>Клиент:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="clientBox">
                 <property name="maximumSize">
                  <size>
                   <width>319</width>
                   <height>16777215</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,2">
               <item>
                <widget class="QLabel" name="objectLabel">
                 <property name="text">
                  <string>Объект:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="objectBox"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,2">
               <item>
                <widget class="QLabel" name="apartmentLabel">
                 <property name="text">
                  <string>Квартира:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="apartmentBox"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_23" stretch="1,2">
               <property name="spacing">
                <number>7</number>
               </property>
               <item>
                <widget class="QLabel" name="costLabel">
                 <property name="text">
                  <string>Стоимость:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="costEdit">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                 <property name="placeholderText">
                  <string>0</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,2">
               <item>
                <widget class="QLabel" name="discountLabel">
                 <property name="text">
                  <string>Скидка:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="discountEdit">
                 <property name="placeholderText">
                  <string>0</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="1,2">
               <item>
                <widget class="QLabel" name="totalLabel">
                 <property name="text">
                  <string>Итого:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="totalEdit">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                 <property name="placeholderText">
                  <string>0</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="1,2">
                 <item>
                  <widget class="QLabel" name="paymentTypeLabel">
                   <property name="text">
                    <string>Тип оплаты:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="paymentTypeBox"/>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="1,2">
                 <item>
                  <widget class="QLabel" name="currencyLabel">
                   <property name="text">
                    <string>Валюта:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="currencyEdit">
                   <property name="readOnly">
                    <bool>true</bool>
                   </property>
                   <property name="placeholderText">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_20" stretch="1,2">
                 <item>
                  <widget class="QLabel" name="installmentLabel">
                   <property name="text">
                    <string>Рассрочка:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QCheckBox" name="installmentCheckBox">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_14" stretch="1,2">
                 <property name="spacing">
                  <number>7</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="initialPaymentLabel">
                   <property name="text">
                    <string>Первоначальный взнос:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="initialPaymentEdit">
                   <property name="enabled">
                    <bool>true</bool>
                   </property>
                   <property name="placeholderText">
                    <string>0</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_22" stretch="1,2">
                 <property name="spacing">
                  <number>7</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="monthQuantityLabel">
                   <property name="text">
                    <string>Сколько месяцев:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="monthQuantityEdit">
                   <property name="readOnly">
                    <bool>false</bool>
                   </property>
                   <property name="placeholderText">
                    <string>0</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_15" stretch="1,2">
                 <property name="spacing">
                  <number>7</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="monthlyPaymentLabel">
                   <property name="text">
                    <string>Каждый месяц:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="monthlyPaymentEdit">
                   <property name="readOnly">
                    <bool>true</bool>
                   </property>
                   <property name="placeholderText">
                    <string>0</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="descriptionFrame">
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QLabel" name="descriptionLabel">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Описание:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="descriptionEdit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>150</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="addButton">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>52</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Сохранить</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
